src/CoreBundle/Resources/doc/backend-assets.rst

```diff
@@ -0,0 +1,20 @@
+Backend Assets
+==============
+
+CoreUI Theme
+------------
+
+The backend uses the CoreUI theme. To make changes e.g. to SCSS variables, proceed as follows:
+
+- Install Node.js and NPM on your machine.
+- Sources are located under `src/CoreBundle/Resources/asset-source/coreui-2.1.6`. Make changes there.
+- In a terminal, go to `src/CoreBundle/Resources/asset-source/coreui-2.1.6`.
+- Run `npm install` (ignore dependency warnings - we manage dependencies separately).
+- Run `npm run dist`.
```

Comment:
(Maybe here or for future dependencies: I have provided a shell script for similar installation steps which does these steps: call run (= compile), copy all relevant files to relevant directories.)

Reply:
I thought about adding a build script but decided against it, for two reasons:
1. I didn't want to make assumptions on the developer's OS, and so didn't want to just build a bash script.
2. Also I didn't want to write a simplistic script that contains just the commands without proper guidance and error handling (leading to a cascade of errors when the first command fails).

Doing it "right" would cost too much time now - it should be quite a rare case to change the sources, so I think it's OK to do this manually.